tests-e2e/argocd/argocd_instance_test.go

```diff
@@ -87,23 +87,7 @@ var _ = Describe("Standalone ArgoCD instance E2E tests", func() {
 			Expect(err).ToNot(HaveOccurred())
 
 			By("creating ArgoCD application")
-			app := appv1.Application{
-				ObjectMeta: metav1.ObjectMeta{
-					Name:      "argo-app-6",
-					Namespace: argocdNamespace,
-				},
-				Spec: appv1.ApplicationSpec{
-					Source: appv1.ApplicationSource{
-						RepoURL:        "https://github.com/redhat-appstudio/managed-gitops",
-						Path:           "resources/test-data/sample-gitops-repository/environments/overlays/dev",
-						TargetRevision: "HEAD",
-					},
-					Destination: appv1.ApplicationDestination{
-						Name:      argocdv1.ClusterSecretName,
-						Namespace: destinationNamespace.Name,
-					},
-				},
-			}
+			app := appFixture.BuildArgoCDApplication("argo-app-01", argocdNamespace, fixture.RepoURL, fixture.GitopsDeploymentPath, "HEAD", "", "", argocdv1.ClusterSecretName, destinationNamespace.Name, &appv1.SyncPolicyAutomated{})
```

Comment:
As above, for the calls to BuildArgoCDApplication, IMHO it's a bit more elegant in this case to leave it as a Go obj.